tests/test-class-wpseo-metabox.php

```diff
@@ -91,6 +91,40 @@ public function test_column_heading_has_metadesc() {
 	}
 
 	/**
+	 * @covers WPSEO_Metabox::column_hidden
```

Comment:
What does this exactly test? I'm seeing at least 2 tests in this function. Separate them and give them clear names so we can tell by the test name what it tests. For example, `test_column_hidden_NO_HIDDEN_columns()` and `test_column_hidden_ARE_NOT_hidden()`. For more ideas/inspiration, see these (premium) tests: https://github.com/Yoast/wordpress-seo-premium/blob/trunk/tests/premium/test-class-post-watcher.php